api/external/nodes/manager/manager.proto

```diff
@@ -169,62 +246,141 @@ service NodeManagerService {
 message ConnectResponse{}
 
 message Id {
+	option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+		json_schema: {
+			required: ["id"]
+		}
+		example: {value: '{"uuid": "cd3ad3d9-2776-4ef1-a904-4c229d1642ee"}'}
+	};
+	// UUID for the node manager.
 	string id = 1;
 }
 message Ids {
+	// List of node manager UUIDs.
 	repeated Id ids = 1;
 }
 
 message Query {
+	option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+		json_schema: {
+			required: ["filter_map"]
+			}
+		example: {value: '{"filter_map": [{"key": "manager_type","values":["aws-ec2"]}],"sort": "date_added"}'}
+	};
+	// Filters for the query: "manager_type".
 	repeated chef.automate.domain.compliance.api.common.Filter filter_map = 25;
-
+	// Sort the results in ascending or descending order.
 	enum OrderType {
 		ASC = 0;
 		DESC = 1;
 	}
 	OrderType order = 21;
+	// Field to use for sorting.
+	// Valid fields are: name, type, status, status_message, date_added.
 	string sort = 22;
+	// Starting page for the list. For example, if your query returns 100 pages,
+	// and you know you're looking for a node manager somewhere in the middle,
+	// you might want to start on 50.
 	int32 page = 23;
+	// Number of results on each page.
 	int32 per_page = 24;
 }
 
 message Fields {
+	option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+		json_schema: {
+			required: ["fields"]
```

Comment:
this is a response message -- i'm not sure "required" fields make sense for response messages?